### DISCUSSION TITLE
Support "start" and "end" text-alignment by default

### DISCUSSION THREAD

**Discussion by trm217** [tailwindlabs/tailwindcss#5325] [Ideas]
As seen in the TailwindCSS docs, the following text-alignments are supported by default:
<img width="574" alt="Screenshot 2021-08-28 at 12 36 24" src="https://user-images.githubusercontent.com/30313631/131215221-f33bc2b8-cb36-4ddb-9f7c-b0d53f33af68.png">

I believe it would be a good idea to support the "start" and "end" alignment by default as well.
Unlike "left" and "right" alignment, "start" and "end" adapt as expected when setting rtl (right-to-left) on the html-document.
(As described in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#values)).
<img width="651" alt="Screenshot 2021-08-28 at 12 38 32" src="https://user-images.githubusercontent.com/30313631/131215278-7e8ae65a-ad02-4a1d-a178-81b70117d9f7.png">

Since localization is a concern for many applications I believe this would be a good addition to the default text-alignment options.
What are your thoughts on this matter?

**Comment by Sulaiman-sa** [comment]
It'd be nice if this was added.
You can also check this plugin, which does just that https://www.npmjs.com/package/tailwindcss-rtl
